data/locale/ar.ts

```diff
@@ -6308,7 +6308,7 @@ Remember to also save your project manually. You can choose to disable saving wh
         <translation type="unfinished"/>
     </message>
     <message>
-        <source>The following errors occured while loading: </source>
+        <source>The following errors occurred while loading: </source>
```

Comment:
The locale files are updated automatically when a string in the source code is changed via our translation project by happy little digital elves. Please revert the changes in `data/locale/`.